src/store/modules/accounts/actions.js

```diff
@@ -15,31 +18,46 @@ const { startLoading, endLoading } = createActionHelpers({
  */
 export default {
   /**
-   * Template
+   * get the account details based on an address
+   * @param {Object} state
    * @param {Function} commit
    * @param {Function} dispatch
-   * @param {*} payload
-   * @return {Promise<any>}
+   * @param {String} address
+   * @return {*}
    */
-  template: function ({ commit, dispatch }, payload) {
-    /**
-     * start load state
-     */
-    startLoading(dispatch, 'blocks/template')
-
-    /**
-     * Return New Promise
-     */
-    return new Promise((resolve, reject) => {
-      /**
-       * End Loading State
-       */
-      endLoading(dispatch, 'blocks/template')
-
-      /**
-       * Resolve template
-       */
-      return resolve('template')
-    })
+  async get ({ state, commit, dispatch }, address) {
+    startLoading(dispatch, 'accounts/get')
+
+    const client = await ae
+
+    const [{ balance }, { transactions }] = await Promise.all([
+      await client.api.getAccountBalance(address),
```

Comment:
If you are using `await` here, it means that `Promise.all` will receive an array of actual values instead of promises, so `Promise.all` has no sense in this case.

Reply:
Will update this. Though this was legacy code, that i copy/pasted to have an idea of what to get from it.